stories/accordion/index.stories.mdx

```diff
@@ -0,0 +1,141 @@
+import {
+  ArgsTable,
+  Meta,
+  Story,
+  Canvas,
+  Description,
+  Primary,
+} from '@storybook/addon-docs/blocks';
+import { useState } from 'react';
+import { text, boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { Accordion, Typography } from 'src/shared-components';
+import { StandardAccordion } from './standardAccordion.stories';
+import { TruncatedAccordion } from './truncatedAccordion.stories';
+import { DisabledAccordion } from './disabledAccordion.stories';
+import { NoBorderAccordion } from './noBorderAccordion.stories';
+import { RightAlignedArrowAccordion } from './rightAlignedArrowAccordion.stories';
+import { ThumbnailsAccordion } from './thumbnailsAccordion.stories';
+import { useAccordionState } from './utils';
+import accountImage from 'shared/account.png';
+import personImage from 'shared/person.jpg';
+
+import * as AccordionStories from './index.stories.tsx';
```

Comment:
For more context on this convention: https://github.com/storybookjs/storybook/pull/11752